src/transform.js

```diff
@@ -171,6 +172,14 @@ export default function (stream, {validate = true, fix = true}) {
 			}
 		}
 
+		function handle003() {
+			marcRecord.fields = marcRecord.fields.filter(field => field.tag !== '003');
```

Comment:
Better to do it like this thank to re-assign: `marcRecord.get(/^003$/).forEach(f => marcRecord.removeField(f));`

Reply:
https://github.com/NatLibFi/marc-record-js why there is no removeField() example in here?